serotiny/models/vae/base_vae.py

```diff
@@ -154,12 +154,9 @@ def forward(self, batch, decode=False, compute_loss=False, **kwargs):
         if not compute_loss:
             return x_hat, z_parts, z_parts_params, z_composed
 
-        (
-            loss,
-            reconstruction_loss,
-            kld_loss,
-            kld_per_part,
-        ) = self.calculate_elbo(batch, x_hat, z_parts_params)
+        (loss, reconstruction_loss, kld_loss, kld_per_part,) = self.calculate_elbo(
```

Comment:
we should make sure we're all using the same linters so these types of changes are minimized